package.json

```diff
@@ -2,13 +2,15 @@
   "name": "npm-check",
   "version": "5.1.3",
   "description": "Check for outdated, incorrect, and unused dependencies.",
-  "main": "lib",
+  "main": "bin",
```

Comment:
`bin` should only be used when running from the command line, as it requires `cli`. When using `require('npm-check')` it needs to use `lib/index`.